unison-src/builtin-tests/base.md

```diff
@@ -1,3 +1,8 @@
+When this file is modified, CI will create a new codebase and re-run this;
+otherwise it may reuse a previously cached codebase.
+
+Thus, make sure the contents of this file define the contents of the cache
```

Comment:
The contents of this file seem to violate this warning, no?

Reply:
Good catch. I don't know what to change the file to that works though yet, so I reverted my initial change, left the warning, which is still valid even though the file violates the rule, and plan to address in a later PR.